fhirAssembly/src/test/java/io/vantiq/extsrc/fhirAssembly/TestFhirAssembly.java

```diff
@@ -710,23 +722,274 @@ public void test310UpdateEtag() throws Exception {
     }
     
     @Test
-    public void test400Delete() throws Exception {
+    public void test320Patch() throws Exception {
```

Comment:
Here, we'll make BatMan a twin or triplet.  And remove any means to communicate.  I suppose I could have changed the communication to the Bat signal, but doing something different made a better test.